src/shell/components/global-sidebar/GlobalSidebar.js

```diff
@@ -57,155 +106,160 @@ export default connect((state) => {
-            sx={{
-              borderColor: "grey.600",
-              borderStyle: "solid",
-              borderWidth: "1px",
-              backgroundColor: "grey.900",
-              float: "right",
-              mr: -1.5,
-              zIndex: 50,
-              width: "24px",
-              height: "24px",
+        <Box sx={{ px: 2.5, pb: 0, pt: 2.5 }}>
+          {props.openNav ? (
+            <Box
+              component="img"
+              data-src={fullZestyLogo}
+              src={fullZestyLogo}
+              sx={{
+                width: "84.17px",
+                height: "24px",
+              }}
+            />
+          ) : (
+            <Box
+              component="img"
+              data-src={zestyLogo}
+              src={zestyLogo}
+              sx={{
+                width: "24px",
+                height: "24px",
+              }}
+            />
+          )}
+        </Box>
+        <IconButton
+          onClick={props.onClick}
+          sx={{
+            borderColor: "grey.600",
+            borderStyle: "solid",
+            borderWidth: "1px",
+            backgroundColor: "grey.900",
+            float: "right",
+            mr: -1.5,
+            zIndex: 50,
+            width: "24px",
+            height: "24px",
 
-              "&:hover": {
-                backgroundColor: "primary.main",
-                borderColor: "common.white",
+            "&:hover": {
+              backgroundColor: "primary.main",
+              borderColor: "common.white",
 
-                ".MuiSvgIcon-root": {
-                  color: "common.white",
-                },
+              ".MuiSvgIcon-root": {
+                color: "common.white",
               },
-            }}
-          >
-            {props.openNav ? (
-              <KeyboardDoubleArrowLeftIcon
-                fontSize="small"
-                sx={{
-                  color: "grey.600",
-                }}
-              />
-            ) : (
-              <KeyboardDoubleArrowRightIcon
-                fontSize="small"
-                sx={{ color: "grey.600" }}
-              />
-            )}
-          </IconButton>
-          <GlobalMenu openNav={props.ui.openNav} />
-          <GlobalCustomApps openNav={props.ui.openNav} />
-          {/* <GlobalActions hash={props.instance.randomHashID} /> */}
+            },
+          }}
+        >
+          {props.openNav ? (
+            <KeyboardDoubleArrowLeftIcon
+              fontSize="small"
+              sx={{
+                color: "grey.600",
+              }}
+            />
+          ) : (
+            <KeyboardDoubleArrowRightIcon
+              fontSize="small"
+              sx={{ color: "grey.600" }}
+            />
+          )}
+        </IconButton>
+        <GlobalMenu openNav={props.ui.openNav} />
+        <GlobalCustomApps openNav={props.ui.openNav} />
+        <OnboardingCallSection
+          openNav={props.ui.openNav}
+          createdAt={instanceCreationDate}
```

Comment:
I just thought that the parent component looks cleaner if it doesn't have enough conditions. But your argument makes sense as well since it's easier to see the condition in the parent component.

Refactored on latest commit.